### DISCUSSION TITLE
Generate several tessellations with the same statistical properties

### DISCUSSION THREAD

**Discussion by ChaymaeHarfoush** [neperfepx/neper#175] [Q&A] [answered]
I'm trying to get tessellation using any distribution  for example many times voronoi (standard Poisson-Voronoi tessellation) distribution  and they give the same tesstelation files every time I run the command
Which doesn't make sense as a distribution should give every time a new grain sizes
command used: `neper -T -n 100 -morpho "voronoi" -dim 2`
the same problem is repeated while using ‘graingrowth’ for morpho argument
command used: `neper -T -n from_morpho -morpho "diameq:lognormal(0.1,0.03),1-circularity:lognormal(0.145,0.03)" -dim 2 -o last_last`

I am not sure where is the problem, **I need to generate different grains shape and size from the same mean and SD**

**Comment by rquey** [comment, marked as answer]
Use the `-id` option to get different tessellations: `neper -T -n 100 -morpho "voronoi" -dim 2 -id 1` and `neper -T -n 100 -morpho "voronoi" -dim 2 -id 2` will be different.
